master/db.ffnord

```diff
@@ -29,6 +29,10 @@ gw02  A     10.187.1.2
 1.ntp.services  AAAA  2a03:2267::101	;ffhh srv01
 2.ntp.services  AAAA	2a03:2267::102	;ffhh srv02
 
+; Knotenforumlar
+formular.ffnord	CNAME	srv01
+formular		CNAME	formular.ffnord
```

Comment:
that was the first time I edited a bind db file.

I `push --amend`ed a new version